adviser/base/argo-workflows/trigger-integration.yaml

```diff
@@ -14,9 +14,9 @@ spec:
           - name: "THOTH_ADVISER_METADATA"
       outputs:
         parameters:
-          - name: origin
+          - name: metadata_origin
```

Comment:
does adding a default value fix the issue?

Reply:
https://github.com/thoth-station/workflow-helpers/pull/67#issuecomment-691073150